career/expectations/software-development-analyst/README.md

```diff
@@ -0,0 +1,130 @@
+
+### Hard Skills
+
+- **Is fluent in most of the tech stack that we use** (Ruby on Rails, React, Next.js, TypeScript, Python, GraphQL and others).
+- **Is fluent in TDD, understands the importance of it and propagates it to the team**.
+- Is capable of writing tests that isolate application layers in a meaningful way (both on backend and frontend), adding value to our test pyramid.
+- Is capable of writing code according to our [Styleguide](https://github.com/GeekHunter-Brasil/hello-world) and improving upon it (making new suggestions and such).
+- Is fluent with common database technologies. Is capable of solving complex performance and/or modeling problems (e.g. optimizing a slow query by re-implementing it or changing the database structure).
+- Has solid experience with other database technologies (e.g. non-relational databases, graph databases, etc). Knows how and when to apply those options in daily tasks.
+- Is fluent in git. Can solve just about any problem that may arise in a repo and is capable of using different strategies to improve the development flow (e.g. use squash and rebase to preserve a linear history)
+- Is fluent in Object Oriented Programming, knows the key points to it and is able to write high-quality OOP code.
+- Has solid experience with Functional Programming, knows advantages and disadvantages to each paradigm.
+- Has solid experience with concepts such as data immutability, pure functions, reactivity.
+- **Has solid experience in software architecture**. Is able to implement common patterns in a disorganized repository.
+- Has solid experience in application design and is capable of choosing a good design according to the complexity or challenge (e.g. monoliths, micro-services, serveless, etc).
+- **Is fluent in debugging code** and can debug just about any problem that may arise, deeply understanding what is happening and what the problem is.
+- **Is fluent at debugging external code** (such as a dependency), and can easily fix it via a pull request or using an isolated version if needed.
+- Is capable of solving any problem that doesn't need R&D (Research & Development) efficiently.
+- Is capable of working in problems that need R&D conducting the research in an effective way, prioritizing the use of their own time in the best way.
+
+### Soft Skills
+
+- Is able to keep their presence in synchronous (e.g. squad rituals such as dailies, plannings, reviews; pair programmings) and asynchronous moments (e.g. discussions on Slack or Basecamp).
+- Is really motivated to learn more and develop their skills.
+- **Has experience in leadership positions, mentoring other people** (usually in technical aspects).
+- Understands their own strong or weak points, and sees something they want to prioritize in their career. The leadership process here is usually easier, since **this person is able to develop their skills with a high level of autonomy**.
```

Comment:
```suggestion
- Understands their own strengths and weaknesses, and sees something they want to prioritize in their career. The leadership process here is usually easier, since **this person is able to develop their skills with a high level of autonomy**.
```